extensions/wikia/WallNotifications/tests/WallNotificationEntityTest.php

```diff
@@ -203,15 +230,11 @@ private function getTitleMock( array $params ) {
 		return $title;
 	}
 
-	private function getParentWallMessageMock( array $params ) {
-		if ( empty( $params['parentWallMessage']['id'] ) ) {
-			return null;
-		}
+	private function getParentWallMessageMock( array $params ): PHPUnit_Framework_MockObject_MockObject {
+		$parentTitleMock = $this->getParentTitleMock( $params );
+		$parentUserMock = $this->getParentUserMock( $params );
 
-		$parentTitle = $this->getParentTitleMock( $params );
-		$parentUser = $this->getParentUserMock( $params );
-
-		$parentWm = $this->getMockBuilder( 'WallMessage' )
+		$parentWallMessageMock = $this->getMockBuilder( 'WallMessage' )
 			->setMethods( [
 				'load', // should do nothing
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)